<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeliveryQueue</class>
 <widget class="QMainWindow" name="DeliveryQueue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delivery Queue</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Queue</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QTreeView" name="treeView">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblFilterPattern">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ledFilterPattern">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblFilterColumn">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search by column:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cboFilterColumn">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Transmission #</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Required Delivery Date</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Location</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ship Name/Hull #</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ECN/TECN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Transit Method</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Classification</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string># of Items</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Media Type</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Staffing Level</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Required Ship Date</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Required Start Date</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="chkFilterCS">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Case sensitive filtering</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuDelivery">
    <property name="title">
     <string>&amp;Delivery</string>
    </property>
    <widget class="QMenu" name="menuNewDelivery">
     <property name="title">
      <string>&amp;New Delivery</string>
     </property>
     <addaction name="actAddManually"/>
     <addaction name="actImport"/>
    </widget>
    <addaction name="menuNewDelivery"/>
    <addaction name="actEditDelivery"/>
    <addaction name="actDeleteDelivery"/>
    <addaction name="actArchiveDelivery"/>
    <addaction name="actChangeSchedule"/>
   </widget>
   <addaction name="menuDelivery"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actRefreshQueue"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actEditDelivery">
   <property name="text">
    <string>&amp;Edit Delivery</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actDeleteDelivery">
   <property name="text">
    <string>&amp;Delete Delivery</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actArchiveDelivery">
   <property name="text">
    <string>&amp;Archive Delivery</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actRefreshQueue">
   <property name="text">
    <string>&amp;Refresh Queue</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actAddManually">
   <property name="text">
    <string>Add &amp;Manually</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actImport">
   <property name="text">
    <string>&amp;Import from Excel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionweq">
   <property name="text">
    <string>weq</string>
   </property>
  </action>
  <action name="actChangeSchedule">
   <property name="text">
    <string>Change Schedule</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>ledFilterPattern</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
